samples/user-interface/appwidgets/src/main/java/com/example/platform/ui/appwidgets/glance/weather/WeatherRepo.kt

```diff
@@ -18,24 +18,30 @@ package com.example.platform.ui.appwidgets.glance.weather
 
 import android.os.Build
 import androidx.annotation.RequiresApi
+import androidx.compose.runtime.MutableState
 import com.example.platform.ui.appwidgets.R
 import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
 import java.time.Instant
 import java.time.ZoneId
 import kotlin.random.Random
 
 @RequiresApi(Build.VERSION_CODES.O)
 object WeatherRepo {
 
+    private var _currentWeather = MutableStateFlow(WeatherInfo.Loading as WeatherInfo)
```

Comment:
Nit: it's better to do `MutableStateFlow<WeatherInfo>(WeatherInfo.Loading)` instead of casting

Reply:
Thanks! I still trip on kotlin sometimes